docs/source/user-guide/general/export_save_results.rst

```diff
@@ -0,0 +1,155 @@
+=================================
+Exporting Deepchecks' Results
+=================================
+
+This guide will help you understand how to export deepchecks results, 
+which enables saving them for later viewing, sharing the analysis results as a report,
+sending to different tools, or for further processing steps 
+(e.g. in ci/cd for deciding whether a new model is good enough for deployment).
+
+We will will discuss the two main export options, that are relevant for saving Check or Suite results:
+
+- :ref:`Saving an HTML File <export_save_results__save_result_to_html>` - saving the interactive report as a file
+- :ref:`Exporting to JSON <export_save_results__export_to_json>` - exporting to a serializeable object that can
+  later be used in other Python scopes, and enables processing the checks' output with code.
+
+For sending the results to other tools (e.g. experiment trackers),
+you can either use one of the above send the desires results or report, or check out
+our :ref:`integrations examples section <user_guide__integrations>` in the user guide.
+
+
+.. _export_save_results__save_result_to_html:
+
+Save Result to an HTML Report: ``save_as_html()``
+====================================================
+
+This method saves the Check or Suite's output as an HTML file.
+It expects an output filename or a full file path to the report's destination folder.
+If no name given, the default filename will be 'output.html'.
+
+
+Save as HTML
+--------------
+
+.. code-block:: python
+
+   result.save_as_html('my_results.html')
+   
+
+Save to in-memory buffer
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The suite output can also be written into a file buffers. This can be done by
+setting the file argument with a ``StringIO`` or ``BytesIO`` buffer object.
+
+
+.. code-block:: python
+
+    import io
+
+    html_out = io.StringIO()
+    suite_result.save_as_html(file=html_out)
```

Comment:
I don't think this is necessary

Reply:
agree :-) deleted... (origin was in the original guide)